docker/mms_docker_cpu/wsgi.py

```diff
@@ -1,21 +1,40 @@
 from mms import mxnet_model_server
+import sys
+import os
 
-# Read arguments
-config_file = 'mms_app.conf'
+mms_arg_header = 'MMS Argument'
+config_file = os.environ['MXNET_MODEL_SERVER_CONFIG']
 args = []
-gunicorn_arg_start = '# Gunicorn arguments'
-with open(config_file) as f:
-    content = f.readlines()
-    content = [line.rstrip() for line in content]
-for i, line in enumerate(content):
-    if line == gunicorn_arg_start:
-        break
-    if line.startswith('--') and content[i + 1] != 'optional':
-        args.append(line)
-        if line == '--models':
-            args += content[i + 1].split(' ')
-        else:
-            args.append(content[i + 1])
+found_mms_args = 0
+
+print("Using %s for config file" % config_file)
```

Comment:
All print and user facing comments should be revisited to make it more standardized, easy and interactive.

Reply:
Re-did this. We will need to revisit this.